completion/available/apm.completion.bash

```diff
@@ -24,54 +24,59 @@
 
 # repo: https://github.com/vigo/apm-bash-completion
 
-__apm(){
-    local cur prev options apm_command
-    COMPREPLY=()
-    cur="${COMP_WORDS[COMP_CWORD]}"
-    prev="${COMP_WORDS[COMP_CWORD-1]}"
-    __apm_get_command
-    if [[ $cur = -* ]]; then
-        options="--color"
-        if [[ -z $apm_command ]]; then
-            options="$options --version --help"
+if _command_exists apm
```

Comment:
How do we want to handle sources that are copied from another repo? Should we reformat these as well? If we do, it will make updating a lot more difficult - see the _vendor_ discussion...

Reply:
@nwinkler I didn't get what you are saying

Reply:
> see the vendor discussion

May I have the link?

Reply:
#1776 is the issue I was referring to.

What I was trying to say: It looks like the `apm` completion was copied from [another repo](https://github.com/vigo/apm-bash-completion) into Bash-it. If we reformat this file here, it will become more difficult to copy in new versions from the source repo, since we'll then have to reformat the new version again. Doing a `diff` will be more difficult...

Reply:
@nwinkler Can i have the list of other completion files so that I can revert them all it once?

Reply:
Sorry, I don't have any list of that kind. The header in this file has a reference to another repo, that's what I noticed. Maybe go through the changed files and take a look whether they refer to another repo...

Reply:
This is done